openlane/steps/klayout.py

```diff
@@ -285,6 +290,12 @@ def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
 
         kwargs, env = self.extract_env(kwargs)
 
+        tile_size_options = []
+        if tile_size := self.config["KLAYOUT_XOR_TILE_SIZE"]:
+            tile_size_options += ["--tile-size", str(tile_size)]
+
+        thread_count = self.config["KLAYOUT_XOR_THREADS"] or os.cpu_count() or 1
+
```

Comment:
Can you add a warning here similar to https://github.com/efabless/openlane2/blob/fbfe20ebf95e1b3e9230821ea9c6d9b6f7ef9989/openlane/steps/openroad.py#L843

Reply:
It's not really a warning *per se* (warnings are more for things that are unrecommended), but we can add an "info" call

Reply:
Sure. Variables or arguments set by OL which aren't saved with the step state should be printed at least.